exercises/practice/word-search/.meta/example.cairo

```diff
@@ -0,0 +1,317 @@
+#[derive(Debug, PartialEq, Drop)]
+pub struct SearchResult {
+    pub word: ByteArray,
+    pub start_col: usize,
```

Comment:
tip: having multiple fields (in this case two) with the same suffixes like `_col` and `_row` tells you that you can logically group these in a struct, making it clearer what the purpose is. 

```rust
#[derive(Drop, Debug, PartialEq)]
struct Position {
    col: usize,
    row: usize,
}

#[derive(Debug, PartialEq, Drop)]
pub struct SearchResult {
    pub word: ByteArray,
    pub start: Position,
    pub end: Position,
}
```

You could even use a tuple instead of a struct:

```rust
#[derive(Debug, PartialEq, Drop)]
pub struct SearchResult {
    pub word: ByteArray,
    pub start: (usize, usize),
    pub end: (usize, usize),
}
```

It's shorter, but a bit less clear.

You don't have to change this, it's really just a tip